lib/Doctrine/Persistence/Mapping/Driver/MappingDriver.php

```diff
@@ -13,6 +13,7 @@ interface MappingDriver
      * Loads the metadata for the specified class into the provided container.
      *
      * @param string $className
+     * @psalm-param class-string $className
```

Comment:
I guess here `@template T of object` could be used to specify `class-string<T>` and `ClassMetadata<T>`.

Reply:
I don't really understand how I can fix the resulting error, maybe I should ignore it…

Reply:
hmmm I guess so, what I don't know is why it didn't complain calling `object::loadMetadata` 🤔

Reply:
https://github.com/phpstan/phpstan/issues/5023